pkg/workflow/tasks/discover.go

```diff
@@ -336,7 +336,7 @@ func NewViewTaskDiscover(pd *packages.PackageDiscover, cli client.Client, cfg *r
 	handlerProviders := providers.NewProviders()
 
 	// install builtin provider
-	query.Install(handlerProviders, cli, cfg)
+	query.Install(handlerProviders, cli, cfg, nil)
```

Comment:
Is the context factory nil?

Reply:
ctxFactory nil means it will use context.Background().

It is used to enforce identity impersonation when using in Application Workflow.

Reply:
Here means executing velaQL from API without identity impersonation?

Reply:
Yes.